src/draw/nxp/pxp/lv_pxp_osa.c

```diff
@@ -25,6 +25,11 @@
     #include "semphr.h"
 #endif
 
+#if defined(__ZEPHYR__)
```

Comment:
Remove the above inclusions:
```
#if defined(SDK_OS_FREE_RTOS)
    #include "FreeRTOS.h"
    #include "semphr.h"
#endif
```

and replace it with:
```
#include "../../../osal/lv_os.h"
```

Reply:
Applied your suggestion.

Reply:
where?

Reply:
Just need the include and then the PR can be merged.

Reply:
I suppose that since the PR was merged once GitHub does not display new changes. I guess a new PR is needed